pom.xml

```diff
@@ -121,7 +121,7 @@
     <dependency>
       <groupId>org.apache.tika</groupId>
       <artifactId>tika-parsers</artifactId>
-      <version>1.25</version>
+      <version>1.17</version>
```

Comment:
Hi Richard, thanks for the review. 
Yes, I sent the why on the email, maybe I should added here too :P

I marked the PR as draft since a couple of Jakarta EE links needs to be updated along with dates and  sha's